### PR TITLE
daily case trend graph palau

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
[#2104](https://github.com/beyondessential/tupaia-backlog/issues/2104)
[#2105](https://github.com/beyondessential/tupaia-backlog/issues/2105)

### CODE REVIEW

File: packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  Conjunctivitis: {
+    dataElement: 'PSSS_CON_Daily_Cases',
+    name: 'Conjunctivitis',
+    color: '#BE72E0', // pink
+  },
+};
+
+const getDashboardReportId = thingName => `PSSS_NR_${thingName}_Daily_Case_Trend_Graph_Country`;
+
+const dashboardGroupCode = 'NR_PSSS_Syndromic_Surveillance';
```
Comment (chris-bes) [inline packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js:54]: I thought this dashboard is for Palau, but isn't `NR` stands for Nauru?

File: packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    dataElement: 'PSSS_CON_Daily_Cases',
+    name: 'Conjunctivitis',
+    color: '#BE72E0', // pink
+  },
+};
+
+const DASHBOARD_GROUP_TEMPLATE = {
+  organisationLevel: 'Facility',
+  userGroup: 'PSSS Tupaia',
+  organisationUnitCode: 'NR',
```
Comment (chris-bes) [inline packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js:55]: I thought this dashboard is for Palau, but isn't NR stands for Nauru?

**Review by chris-bes — CHANGES_REQUESTED** [review]
Nice work! Just a few minor suggestions

File: packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js (hunk 2 above)
Comment (billli0) [reply packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js:55]: Thanks Chris! Good spot :pray:

File: packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  return {
+    id,
+    dataBuilder: 'sumPerPeriod',
+    dataBuilderConfig: {
+      dataClasses: {
+        value: {
+          codes: [dataElementCode],
+        },
+      },
+      aggregationType: 'FINAL_EACH_WEEK',
```
Comment (chris-bes) [inline packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js:66]: Sorry @billli0 , I just spot another thing. Isn't these 2 dashboards daily?

**Review by chris-bes — CHANGES_REQUESTED** [review]

File: packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js (hunk 3 above)
Comment (billli0) [reply packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js:66]: Yeah, one for country level, one for facility level

Comment (chris-bes) [reply packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js:66]: But you're aggregating the data weekly...?

Comment (billli0) [reply packages/database/src/migrations/20210414064857-DailyCasesTrendGraphPalauCountryLevel-modifies-data.js]: Thanks Chris!

**Review by chris-bes — APPROVED** [review]
Looks good!

**Pull request merged**